packages/web-fragments/tsconfig.json

```diff
@@ -1,21 +1,11 @@
 {
+	"extends": "./tsconfig.base.json",
 	"compilerOptions": {
-		"target": "ESNext",
-		"useDefineForClassFields": true,
-		"lib": ["DOM", "DOM.Iterable", "ESNext"],
-		"allowJs": true,
-		"skipLibCheck": true,
-		"esModuleInterop": false,
-		"allowSyntheticDefaultImports": true,
-		"strict": true,
-		"forceConsistentCasingInFileNames": true,
-		"module": "ESNext",
-		"moduleResolution": "Bundler",
-		"resolveJsonModule": true,
-		"isolatedModules": true,
+		"declaration": false,
+		"emitDeclarationOnly": false,
```

Comment:
do you need to add this?

if you'd not you could then avoid to have to do the `--emitDeclarationOnly false` in the `web-fragments` package